tests/agent_test.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from typing import Any, Dict, List
+
+import pytest
+import yaml
+from fixieai import FixieClient
+
+
+@pytest.fixture
+def client():
+    client = FixieClient()
+    return client
+
+
+@pytest.fixture
+def session(client):
+    session = client.create_session()
+    return session
+
+
+def read_spec() -> Dict[str, List[Dict[str, Any]]]:
```

Comment:
I think it would be better if this were distributed across directories in individual `agent_test.yaml` files. Makes it harder to forget to update this when adding a new agent, and it would be good for our developers too!

Reply:
Done! I removed some of the tests that were for non-example Agents.